src/wolfgame.py

```diff
@@ -6787,6 +6809,10 @@ def cgamemode(cli, arg):
         cli.msg(chan, "Mode \u0002{0}\u0002 not found.".format(modeargs[0]))
 
 
+def expire_start_votes(cli, chan):
```

Comment:
Missing lock and phase check. Also that message is really awkward, I'd nix the bold at the very least.